misk-feature-testing/src/test/kotlin/misk/feature/testing/FakeFeatureFlagsTest.kt

```diff
@@ -58,19 +57,31 @@ internal class FakeFeatureFlagsTest {
 
   @Test
   fun invalidKeys() {
+    subject.override(FEATURE, 3)
```

Comment:
Previously this test wasn’t exercising what it wanted to exercise. Because the feature wasn’t provided, even if the token was valid these tests would have passed.

```
java.lang.IllegalArgumentException: String flag Feature(name=which-dinosaur) must be overridden with override() before use
	at misk.feature.testing.FakeFeatureFlags.getString(FakeFeatureFlags.kt:41)
	at misk.feature.FeatureFlags$DefaultImpls.getString(FeatureFlags.kt:67)
	at misk.feature.testing.FakeFeatureFlags.getString(FakeFeatureFlags.kt:18)
	at misk.feature.testing.FakeFeatureFlagsTest$invalidKeys$e$1.invoke(FakeFeatureFlagsTest.kt:62)
	at misk.feature.testing.FakeFeatureFlagsTest$invalidKeys$e$1.invoke(FakeFeatureFlagsTest.kt:10)
	at misk.feature.testing.FakeFeatureFlagsTest$inlined$sam$i$org_junit_jupiter_api_function_Executable$0.execute(Assertions.kt)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:54)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:36)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:1179)
```